.gitignore

```diff
@@ -90,4 +90,5 @@ dist/
 .vscode/
 temp/
 .eslintcache
-.env.local
\ No newline at end of file
+.env.local
+.env
```

Comment:
This shouldn't be ignored (if you need to customize the environment use `.env.local`). This is the template for environment variables.

Reply:
@pmigueld updated